src/web/mjs/connectors/BananaMecanica.mjs

```diff
@@ -0,0 +1,13 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class BananaMecanica extends WordPressMadaraNovel {
+
+    constructor() {
+        super();
+        super.id = 'bananamecanica';
+        super.label = 'Banana Mecânica';
+        this.tags = [ 'manga', 'webtoon', 'portuguese', 'scanlation' ];
```

Comment:
Are there `manga` on the website? Seems to me like only `webtoon` is present.
Is this really a `scanlation` group or just another aggregator?

Reply:
Is a Scanlation group of Yaoi, link of a [banner group](https://leitorbm.com/wp-content/uploads/WP-manga/data/manga_5fc7c46f26ef9/e28146e25cd5d4af2fd6fc2c687d108e/00-capa15-e-1_01.jpg), a [manga](https://leitorbm.com/manga/usagi-no-mori/).